config.lua

```diff
@@ -0,0 +1,35 @@
+Config = {}
+
+Config.UseTarget = GetConvar('UseTarget', 'false') == 'true'
+
+Config.ActivePolice = 0 --<< needed police to activate the mission
+
+--Position to Activate the Mission
+Config.MissionMarker =  vector3(960.71197509766, -215.51979064941, 76.2552947998)   --<< Marker to start the Mission
+Config.dealerCoords =  vector4(960.78, -216.25, 75.25, 268.9422)  							--<< place where the NPC stands
+
+--<< coordinates of the truck's spawn (picked randomly)
+Config.VehicleSpawn1 = vector3(-1327.479736328, -86.045326232910, 49.31)
+Config.VehicleSpawn2 = vector3(-2075.888183593, -233.73908996580, 21.10)
+Config.VehicleSpawn3 = vector3(-972.1781616210, -1530.9045410150, 4.890)
+Config.VehicleSpawn4 = vector3(798.18426513672, -1799.8173828125, 29.33)
+Config.VehicleSpawn5 = vector3(1247.0718994141, -344.65634155273, 69.08)

```

Comment:
This would probably be nicer in a table with just a list of vector3's. Would need to be handled differently in the code though.